annotations/component-annotations/src/it/issue-71/src/main/java/io/ap4k/examples/component/Main.java

```diff
@@ -23,7 +23,7 @@
 import io.ap4k.kubernetes.annotation.Env;
 
 @CompositeApplication(name = "hello-world", exposeService = true, envVars = @Env(name = "key1", value = "val1"))
-@Link(name = "hello-world", targetcomponentname = "target", envVars = @Env(name = "key1", value = "val1"))
+@Link(name = "hello-world", componentname = "target", envVars = @Env(name = "key1", value = "val1"))
```

Comment:
Can we use `componentName` instead, please?

Reply:
yes, sure!